### DISCUSSION TITLE
[doc] zustand doc website sidebar not prerender all contents

### DISCUSSION THREAD

**Discussion by tjx666** [pmndrs/zustand#2846] [Bug report]
### Bug Description

Which make it hard to crawl page links.

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/e4043979-c362-4667-892b-dc834eb6c204">

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/d3e9de2f-7f4e-451a-be3c-abbccd07e2aa">

the better way is like jotai:

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/f7462653-1290-454b-b3ec-5455e8bd5566">

### Reproduction Link

https://zustand.docs.pmnd.rs/getting-started/introduction

**Comment by dai-shi** [comment]
You might want to report it in https://github.com/pmndrs/docs.
